albatross-1/README.md

```diff
@@ -0,0 +1,207 @@
+# Albatross Testnet Instructions
+
+## Software Requirements
+
+- Go version v1.14.+
+- Stakebird version : [v0.3.0](https://github.com/public-awesome/stakebird/releases/tag/v0.3.0)
+
+### Install Stakebird
+
+```sh
+> mkdir -p $GOPATH/src/github.com/public-awesome
+> cd $GOPATH/src/github.com/public-awesome
+> git clone https://github.com/public-awesome/stakebird && cd stakebird
+> git checkout v0.3.0
```

Comment:
We don't have a tagged release for this yet... I assume you know this but still leaving the comment. :smile_cat:

Reply:
Yeah :) Assuming it'll be tagged before this becomes a real PR.